ui/app/i18n/appointments/locale_en.json

```diff
@@ -161,6 +161,7 @@
   "LOCATION_DESK": "Registration Desk",
   "LOCATION_OPD": "OPD",
   "LOCATION_EMERGENCY": "Emergency",
+  "LOCATION_RECEPTION_DESK": "Registration Desk",
```

Comment:
@rubailly This is a duplicate of LOCATION_DESK. Please remove the one not being used.

Reply:
Also remove  "APPOINTMENT_SERVICE_EMERGENCY_KEY": "Emergency",